src/style.css

```diff
@@ -1,3 +1,70 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  --blue: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+}
+
+.nav {
+  display: flex;
+  justify-content: center;
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+  position: relative;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  line-height: 60px;
```

Comment:
this value isn't the same as you have at figma. 
You should use `padding: 23px 0;` and font styles from figma to make correct height for your link